client/src/pages/Landing/Landing.js

```diff
@@ -91,49 +84,48 @@ const Landing = () => {
     })
   }
 
+  const window = useWindowDimensions()
+
   return (
     <View style={styles.root}>
       <Image
         source={images.background_landing}
+        alt="Language names background image"
```

Comment:
I think this alt could be improved. I know that that this app doesn't stress accessibility as much as Brink, but it would be nice to be clear when we can. Maybe an alt like, "7000 Languages red background with names of endangered languages." What do you think?

Reply:
agreed, fixed